assets/js/youtube.js

```diff
@@ -0,0 +1,24 @@
+// This consists of all Javascript related to Youtube Functions while working so testing can be conducted separately. This will be combined into a single JS towards the end of the project
```

Comment:
@itsbbea  you please delete this file from your branch as it has now been blended into our new script.js file?